### PR TITLE
Added ability to cancel out link (for social buttons, etc).

### PR DESCRIPTION
**Pull request by dawnerd**
Sometimes we don't need the link to go through, like for twitter intents.

### CODE REVIEW

File: lib/ga-track.js [hunk 1]
```diff
@@ -84,7 +84,9 @@
 
       el.on('click', function(e) {
         $.gaTrack(cat, action, label);
-        if (href && !e.metaKey && e.which === 1) {
+        if (el.data('ga-track-href') === false) {
```
Comment (jgallen23) [inline lib/ga-track.js]: isn't this "false" and won't === false?

Comment (dawnerd) [inline lib/ga-track.js]: jquery evaluates, which I just found out. Can even put json in there and it'll become an object.

Comment (jgallen23) [inline lib/ga-track.js]: wow, thats good to know

**Comment by jgallen23** [comment]
looks good.  go ahead and release

**Pull request merged**